scripts/build.sh

```diff
@@ -1,21 +1,20 @@
 #!/bin/bash
 
+PURPLE="\033[0;35m"
+RESET="\033[0m"
+
 set -e
 
 . "$(dirname "$0")/consts.sh"
 
 CURR_VER_FILE="/tmp/current-server-version"
 
-VERSION=$1
-BUILD=$2
-USE_CACHE=$3
-
 if [ -z "$VERSION" ] || [ "$VERSION" == "latest" ]; then
     VERSION=$(sh "$(dirname "$0")"/get-latest-version.sh)
 fi
 
 if [ -z "$BUILD" ] || [ "$BUILD" == "latest" ]; then
-    BUILD=$(sh "$(dirname "$0")"/get-latest-build.sh "$VERSION")
+    BUILD=$(bash "$(dirname "$0")"/get-latest-build.sh "$VERSION")
```

Comment:
The operator ``==`` is specific to bash and is not supported in sh. The calling script (build.sh) is also executed in bash, see line 1